ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml

```diff
@@ -602,6 +602,72 @@ tests:
     test:
     - chain: cucushift-sdn-migration-ovn
     workflow: cucushift-installer-rehearse-azure-ipi-sdn
+- as: gcp-ipi-baselinecaps-none-amd-f28-destructive
+  cron: 31 22 5 * *
+  steps:
+    cluster_profile: gcp-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASELINE_CAPABILITY_SET: None
+      E2E_RUN_TAGS: '@amd64 and @gcp-ipi and @network-ovnkubernetes and not @fips
+        and not @workloads and not @storage'
+      PERSISTENT_MONITORING: "false"
+      TAG_VERSION: '@4.15'
+    test:
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-gcp-ipi-capability-addtionalcaps
+- as: gcp-ipi-baselinecaps-none-arm-f7
+  cron: 49 5 5,13,21,28 * *
+  steps:
+    cluster_profile: gcp-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASELINE_CAPABILITY_SET: None
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      E2E_RUN_TAGS: '@arm64 and @gcp-ipi and @network-ovnkubernetes and not @fips
+        and not @workloads and not @storage'
+      OCP_ARCH: arm64
+      PERSISTENT_MONITORING: "false"
+      TAG_VERSION: '@4.15'
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-gcp-ipi-capability-addtionalcaps
+- as: gcp-ipi-compact-filestore-csi-amd-f28-destructive
+  cron: 24 6 15 * *
+  steps:
+    cluster_profile: gcp-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    dependency_overrides:
+      OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.14
+    env:
+      E2E_RUN_TAGS: '@amd64 and @gcp-ipi and @network-ovnkubernetes and not @fips
+        and not @storage'
```

Comment:
@Phaow here the job will not execute storage case, is that okay for your team?

Reply:
@jianlinliu Yeah, ok for storage. Since the `cucushift-installer-rehearse-gcp-ipi-filestore-csi` workflow change the default storageClass to filestore csi and for storage csi tests most test cases are in ginkgo so exclude the cucushift cases(`not @storage` only exclude the storage cucushift cases, storage golang cases will still execute) is ok (also for some timeout/size limitation of filestore many cucushift cases are not suitable). Thanks for the double checking!
cc to @chao007 @duanwei33